vision/src/autogluon/vision/object_detection/object_detection.py

```diff
@@ -48,17 +53,18 @@ def fit(self,
             For more details of how to construct a object detection dataset, please checkout:
             `http://preview.d2l.ai/d8/main/object_detection/getting_started.html`.
             If a string is provided, will search for k8 datasets.
-        val_data : pd.DataFrame or str, default = None
-            Training data, can be a dataframe like image dataset.
+        tuning_data : pd.DataFrame or str, default = None
+            Holdout tuning data for validation, can be a dataframe like image dataset.
```

Comment:
"Holdout tuning data for validation" -> "Held-out validation data reserved for model selection and hyperparameter-tuning"